.gitignore

```diff
@@ -1,7 +1,8 @@
 inviwo-log.html
 **/.DS_Store
-tools/doxygen/doc
-tools/doxygen/doc-qt
 *.pyc
 pyconfig.ini
-.idea
\ No newline at end of file
+.idea
+*.qch
```

Comment:
We usually distribute the qch files right? Why should we stop that?

Reply:
Yes, they are located in the doc folder of each module.

Reply:
Do we? If we do, how do we make sure they are recent? 
For me they are listed as untracked files and I thought we would not want to store generated files in the repo? 

Maybe we should then make sure that they are either updated on commit or built when compiling inviwo?

Reply:
We don't want to require you to run all doxygen stuff to get the help in inviwo, hence we distribute them. When one updates docs for a module one should also make sure to update the qch file for that module.

Reply:
Alright, I added them back